### PR TITLE
14798 send form total value overflows to second row moving the incl fee row for fiat

### LINKED ISSUE #14798: Send form "Total" value overflows to second row moving the "Incl. fee" row for fiat
**Describe the bug**
In Send form, the "Total" that is showing value in crypto can overflow to the second row which is for "Incl. fee" for fiat and that also moves the "Review & Send" button.

**Steps to reproduce:**
1. Go to Ethereum send form
2. Enter for example 1$ as an amount
3. See the result on the right with "Review & Send" button

**Info:**
 - Suite version: desktop 24.10.1 (3cbe786b5c8effd03b331faec20a41ef01d6671b)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.10.1 Chrome/126.0.6478.234 Electron/31.6.0 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T2T1 2.8.1 regular (revision 632b9561559b7ab6824bb7eeac072874e07b7b82)
 - Transport: BridgeTransport 3.0.0-bundled.24.10.1

**Expected result:**
Total for crypto and fiat stays on one line.

**Actual result:**
Total value for crypto overflows to second row for "Incl. fee" value

**Screenshots:**
[2024-10-10 13-16-16.webm](https://github.com/user-attachments/assets/605bd8a4-b462-432f-aba0-9f35fe91c104)

### PR DESCRIPTION
**Pull request by seibei-iguchi**
- fixed fees wrapping in send form
- fixed elevation handling in Card and CollapsibleBox components

### Before:
Screenshot 2024-10-17 at 14 16 36

### After:
Screenshot 2024-10-17 at 14 18 16

### CODE REVIEW

File: packages/components/src/components/CollapsibleBox/CollapsibleBox.tsx [hunk 1]
```diff
@@ -174,6 +174,51 @@ export const CollapsibleBox = ({
         setIsOpen(!isOpen);
     };
 
+    const headerContent = (
```
Comment (tomasklim) [inline packages/components/src/components/CollapsibleBox/CollapsibleBox.tsx:177]: How about moving this out of this component `CollapsibleBoxHeader`

File: packages/components/src/components/Card/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    return paddingMap[$paddingType];
+};
+
+export const mapPaddingTypeToPadding = ({ $paddingType }: MapArgs): number | string => {
+    const paddingMap: Record<PaddingType, number | string> = {
+        none: 0,
+        small: spacingsPx.sm,
+        normal: spacingsPx.lg,
+        large: spacingsPx.xl,
```
Comment (tomasklim) [inline packages/components/src/components/Card/utils.tsx:37]: normal -> lg padding
large -> xl padding

How about 
having normal, large and xlarge with proper spacings

Comment (seibei-iguchi) [reply packages/components/src/components/Card/utils.tsx:37]: Sorry, I don't understand the point:)

Comment (tomasklim) [reply packages/components/src/components/Card/utils.tsx:37]: we call paddingType `normal` but we use `lg` spacing.

Comment (seibei-iguchi) [reply packages/components/src/components/Card/utils.tsx:37]: As consumers of the component, we don't care about the right side of the mapping function, that's just internal design system token naming:) For this component, normal means such and such padding. Hope it makes sense:)

**Review by enjojoy — APPROVED** [review]
LGTM 🚀 Tested the change, it works. Fixed card elevation looks good.

File: packages/components/src/components/Card/Card.tsx [hunk 3]
```diff
@@ -103,33 +83,38 @@ const CardContainer = styled.div<
     /* when theme changes from light to dark */
     transition: background 0.3s, box-shadow 0.2s;
 
+    ${({ $fillType, $elevation, theme }) =>
+        $fillType === 'none' &&
```
Comment (peter-sanderson) [inline packages/components/src/components/Card/Card.tsx]: [optional] Not sure if its worth it here, but adding a new fillType wont produce type-error here. So programmer does not know, they need to adjust CSS

```ts
const map: Record<FillType, SomeCSSStyledComponentType> = {
    'none': css`<here is the css for none>`, 
    'default': css`<here is the css for default>`,
}
```

a then do:

```ts
${map[$fillType]}  // <--- here you get type-error if you add new FillType into union
```

File: packages/components/src/components/Card/Card.tsx [hunk 4]
```diff
@@ -103,33 +83,38 @@ const CardContainer = styled.div<
...
+        css`
+            background: none;
```
Comment (peter-sanderson) [inline packages/components/src/components/Card/Card.tsx]: Also this is resetting something from above. I would rather see a explicit styling for every `FillType`  separate.

Comment (peter-sanderson) [reply packages/components/src/components/Card/Card.tsx]: So my logical flow when I read the code is straightforward

File: packages/suite/src/components/backup/BackupSeedCard.tsx [hunk 5]
```diff
@@ -105,7 +105,7 @@ export const BackupSeedCard = ({
     };
 
     return (
-        <Container forceElevation={2} $checked={isChecked} data-testid={dataTest}>
```
Comment (peter-sanderson) [inline packages/suite/src/components/backup/BackupSeedCard.tsx:108]: :clap:

File: packages/suite/src/views/suite/SwitchDevice/CardWithDevice.tsx [hunk 6]
```diff
@@ -64,7 +64,7 @@ export const CardWithDevice = ({
     const { elevation } = useElevation();
 
     return (
-        <Card paddingType="small" forceElevation={0}>
```
Comment (peter-sanderson) [inline packages/suite/src/views/suite/SwitchDevice/CardWithDevice.tsx:67]: :clap: :clap:

**Review by peter-sanderson — APPROVED** [review]
Just that one theoretical/philosophical point about type mapping. We can discuss on the call if you want, but its not a big deal. 

Good job removing that hacky `forceElevation` :tada: 

:shipit:

**Comment by seibei-iguchi** [comment]
> Just that one theoretical/philosophical point about type mapping. We can discuss on the call if you want, but its not a big deal. 
> 
> 
> 
> Good job removing that hacky `forceElevation` :tada: 
> 
> 
> 
> :shipit: 

The force elevation wasn't even doing anything:) If you look at the Card component, it wasn't properly passed.

**Comment by tomasklim** [comment]
Adam when you are into this, could you also check it in trade section? (In case it is really the same) https://github.com/trezor/trezor-suite/issues/14852

**Review by peter-sanderson — APPROVED** [review]

**Pull request merged**